src/Voron/Data/Tables/Table.cs

```diff
@@ -1255,11 +1255,15 @@ public IEnumerable<SeekResult> SeekBackwardFrom(TableSchema.AbstractTreeIndexDef
                 if (it.Seek(last) == false && it.Seek(Slices.AfterAllKeys) == false)
                     yield break;
 
-                it.SetRequiredPrefix(prefix);
-                if (SliceComparer.StartWith(it.CurrentKey, it.RequiredPrefix) == false)
+                if (prefix != null)
                 {
-                    if (it.MovePrev() == false)
-                        yield break;
+                    it.SetRequiredPrefix(prefix.Value);
+
+                    if (SliceComparer.StartWith(it.CurrentKey, it.RequiredPrefix) == false)
+                    {
+                        if (it.MovePrev() == false)
+                            yield break;
+                    }
                 }
```

Comment:
@arekpalinski Can you check that this is alright please?

Reply:
Yes, that is how I expected it will look like